src/rules/declaration-block-no-duplicate-properties/README.md

```diff
@@ -62,6 +62,40 @@ p {
 }
 ```
 
+### `ignore: ["consecutive-duplicates-with-different-values"]`
+
+Ignore consecutive duplicated properties with different walues.
+
```

Comment:
I got confused by the examples below on my first read, so I'd bet other users will, as well. We could provide just a little more explanation to make it very clear that both *consecutive* and *duplicated* are keywords here.